src/main/webapp/app/exam/shared/student-exam-working-time.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, Input, OnInit } from '@angular/core';
```

Comment:
I'm not quite sure what advantage we get using this component over just having a utility function returning the correct difference.

Reply:
The component renders both the time and the difference. That way the difference-calculation-helper does not have to be called everywhere a student’s working time is displayed and it cannot be forgotten.
Using a dedicated component that defines the uniform format how a student’s working time should be rendered seems appropriate.

Reply:
Makes sense, thanks.